README.md

```diff
@@ -1003,6 +1003,48 @@ const MyResponsiveDiv = glamorous.div({
 ```
 </details>
 
+In addition to all this, you can reference a `glamorous` component inside the
+styles of other `glamorous` components (this works because they each implement
+the `toString` method):
+
+<details>
+<summary>Contextual Overrides</summary>
+
+This API is inspired by `styled-components`.
+
+```javascript
+const Link = glamorous.a({
```

Comment:
consider making this opt-in

Reply:
I agree with @kentcdodds, this is annoying 'by default'.